app/src/main/java/org/schabi/newpipe/util/ShareUtils.java

```diff
@@ -79,13 +156,15 @@ private static String getDefaultBrowserPackageName(final Context context) {
      * @param url     the url to share
      */
     public static void shareUrl(final Context context, final String subject, final String url) {
```

Comment:
Shouldn't this be called `shareText`, since it is now used to also share `json`?

Reply:
Yes, I will change method name (a lot files will be changed).